tests/index-test.js

```diff
@@ -25,59 +25,137 @@ describe('rollout', function () {
   })
 
   it('fulfills', function () {
-    subject.handler('secret_feature', {
+    return subject.handler('secret_feature', {
       employee: {
         percentage: 100,
         condition: isCompanyEmail
       }
     })
-    return expect(subject.get('secret_feature', 123, {
-      employee: 'me@company.com'
-    })).to.be.fulfilled
+    .then(function () {
+      var result = subject.get('secret_feature', 123, {
+        employee: 'me@company.com'
+      })
+      return expect(result).to.be.fulfilled
+    })
+  })
+
+  it('fulfills with applicable modifier for percentage', function () {
+    return subject.handler('secret_feature', {
+      everyone: {
+        percentage: 0
+      },
+      employee: {
+        percentage: 100,
+        condition: isCompanyEmail
+      }
+    })
+    .then(function () {
+      var result = subject.get('secret_feature', 123, {
+        employee: 'me@company.com'
+      })
+      return Promise.all([
+        expect(result).to.be.fulfilled,
+        expect(result).to.eventually.equal('employee')
```

Comment:
this is a bit redundant
I believe you can replace the `Promise.all` with `expect(result).to.eventually.equal('employee')`

Reply:
```
      return Promise.all([
        expect(result).to.be.fulfilled,
        expect(result).to.eventually.equal('employee')
      ])
```
could be replaced in other tests also

This appears to work as expected as a replacement
```
return expect(result).to.eventually.equal('employee')
```